### PR TITLE
New Hooks ⚓ 🎈 

### PR DESCRIPTION
**Pull request by mahdiyazdani**
This PR proposes potential and most used internal hooks used across different blocks and extensions for the `sixa` projects.

- `useActiveTab`: Maintains and determines the current state of the active tab.
- `useConditionalRef`: Conditionally return an instance of `useRef`
- `useGetNodeList`: Retrieve list of HTML nodes genearted for each post item.
- `useLatLngBounds`: Get latitude & longitude from address.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
 	"name": "@sixa/wp-react-hooks",
-	"version": "1.2.1",
+	"version": "1.2.2",
```
Comment (kuserich) [inline package.json:3]: Given that we're adding functionality, would it be more sensible if we updated the version to `1.3.0` instead?

**Review by kuserich — APPROVED** [review]
Thank you, @mahdiyazdani . Please also create a release and publish the new version of this package to NPM.

File: package.json (hunk 1 above)
Comment (mahdiyazdani) [reply package.json:3]: Sure, I agree.

**Review by gooklani — APPROVED** [review]
Thank you @mahdiyazdani 🙌

**Pull request merged**